torch/_inductor/runtime/benchmarking.py

```diff
@@ -485,6 +488,15 @@ def benchmark_many_gpu(
         the estimation loop instead of processing a full benchmarking cycle. The
         ranking key is set as a string, instead of a boolean, to ensure lazy benchmarks
         are properly grouped if ranking is enabled.
+        - pruning_key: Similar to `ranking_key` an optional string key that enables
+        pruning. Pruning eliminates slow callables after the stimation loop, and only
+        a subset of the fastest callables are fully benchmarked.
```

Comment:
I don't really understand why pruning_key is a str. should it just be a boolean ? to me a str arg when the semantic is boolean adds additional cognitive complexity.